Makefile

```diff
@@ -1,5 +1,7 @@
 GO ?= go
 
+export GOPROXY=https://proxy.golang.org
```

Comment:
What happens if the proxy is unavailable? Can we set it to something like `https://proxy.golang.org,direct` to still allow direct connections as fallback?

Reply:
That's the default in go v1.13, so I don't think we need to worry about the proxy being unavailable. The change is effectively an early opt-in for go < v1.13.

Regarding the fall back, I don't know if that's valid syntax for GOPROXY. I've checked the docs but didn't see a reference for that. Do you know if that's possible?

Reply:
`invalid $GOPROXY setting: cannot have comma` ... it's really a mutually exclusive setting. If the proxy is unavailable, it will fail.

Reply:
Okay, so the comma separated list seems a go1.13 feature too, LGTM :)